apps/src/aichat/redux/aichatRedux.ts

```diff
@@ -240,13 +240,18 @@ export const onSaveComplete =
     }
 
     changedProperties.forEach(property => {
+      const typedProperty = property as keyof AiCustomizations;
+      const propertiesSpecificityNeeded = ['temperature', 'selectedModelId'];
+      const textSuffix = propertiesSpecificityNeeded.includes(property)
+        ? ` has been updated to ${currentAiCustomizations[typedProperty]}.`
```

Comment:
It'd be slick if we could bold the value as well (I think I remember seeing this in a mock) -- I could imaging changing the type of `chatMessageSuffix` to `ReactNode` and passing in a fragment that has the content bolded. No worries if it's too much of a pain though

Reply:
Chatted offline -> will ask Sam when she's back in office if she wants this specific property boldtyped. If so, wiill update `chatMessageSuffix` to be an object with an optional boldText property. Thanks!

Reply:
Updated and think it definitely looks better. Keep confirm that change is preferred during next triage meeting.

Reply:
@samantha-code is good with the change. Thanks!